src/main/java/net/reini/rabbitmq/cdi/ConnectionConfiguration.java

```diff
@@ -8,29 +13,43 @@
 import java.util.Set;
 import java.util.concurrent.TimeoutException;
 
-import javax.net.ssl.SSLContext;
-
-import com.rabbitmq.client.Address;
-import com.rabbitmq.client.Connection;
-import com.rabbitmq.client.ConnectionFactory;
-
 /**
  * Contains the Rabbit MQ connection configuration
  *
  * @author Patrick Reinhart
  */
-class ConnectionConfiguration implements ConnectionConfig, ConnectionConfigHolder {
+public class ConnectionConfiguration implements ConnectionConfigHolder {
```

Comment:
Only open access if really needed. Is package access not sufficient here?

Reply:
fixed

Reply:
fixed